docs/index.md

```diff
@@ -22,6 +22,7 @@ grid_model/index.md
 grid_features/index.md
 simulation/index
 data/timeseries
+data/datasources
```

Comment:
I think this should be deeper in the index, it's more for developers or expert users. Maybe a "going further" section in grid exchange formats